modin/backends/pandas/query_compiler.py

```diff
@@ -60,6 +60,38 @@ def str_op_builder(df, *args, **kwargs):
     return str_op_builder
 
 
+def _dt_prop_map(property_name):
+    def dt_op_builder(df, *args, **kwargs):
+        dt_s = df.squeeze().dt
+        return getattr(dt_s, property_name).to_frame()
+
+    return dt_op_builder
+
+
+def _dt_map2(property_name):
+    def dt_op_builder(df, *args, **kwargs):
+        dt_s = df.squeeze().dt
+        return getattr(dt_s, property_name)
+
+    return dt_op_builder
+
+
+def _dt_func_map_series(func_name):
+    def dt_op_builder(df, *args, **kwargs):
+        dt_s = df.squeeze().dt
+        return getattr(pandas.Series.dt, func_name)(dt_s, *args, **kwargs).to_frame()
+
+    return dt_op_builder
+
+
+def _dt_func_map(func_name):
```

Comment:
I think this should convert to a `DataFrame` after.

Reply:
This function was added with the same reason. Function used for methods which returns non-pandas types, as this types haven't method `to_frame`.

Reply:
We should be able to standardize all of these into a single function, I believe. All return types should be `pandas.DataFrame` in the function, then different return types should be handled in the API layer (series.py)

Reply:
Tank you! It was fixed.